products/memgraph/v0.51.0/reference_guide/query-modules.md

```diff
@@ -29,11 +29,12 @@ docker run -p 7687:7687 \
 ```
 
 Memgraph will now attempt to load the query modules form all `*.so` and `*.py`
-files it finds in the given directory. Each file corresponds to one query
-module. Names of these files will be mapped to query module names.  So in our
-case, we have an `example.so` which will be mapped to `example` module and a
-`py_example.py` which will be mapped to `py_example` module in the query
-language.
+files it finds in the given directory. The `*.so` modules are usually written
```

Comment:
The `usually` here isn't necessary. `*.so` files will always be written using the C API.